core/lib/contracts/src/lib.rs

```diff
@@ -381,12 +312,6 @@ impl BaseSystemContracts {
         BaseSystemContracts::load_with_bootloader(bootloader_bytecode)
     }
 
-    /// BaseSystemContracts with playground bootloader - used for handling eth_calls.
-    pub fn estimate_gas() -> Self {
-        let bootloader_bytecode = read_bootloader_code("fee_estimate");
-        BaseSystemContracts::load_with_bootloader(bootloader_bytecode)
-    }
```

Comment:
Looks like we have to keep this function and use it in api

Reply:
Do you mean that it is a bug that this function is not used? It's never invoked anywhere.

Reply:
it's supposed to be use for getting bootloader for the latest vm